ui/app/templates/allocations/allocation/task/index.hbs

```diff
@@ -1,11 +1,15 @@
 {{#global-header class="page-header"}}
-  <span class="breadcrumb">Allocations</span>
-  {{#link-to "allocations.allocation" model.allocation class="breadcrumb"}}
-    {{model.allocation.shortId}}
-  {{/link-to}}
-  {{#link-to "allocations.allocation.task" model.allocation model class="breadcrumb"}}
-    {{model.name}}
-  {{/link-to}}
+  <li><a href="#">Allocations</a></li>
```

Comment:
I'm curious, why this rather than `{{link-to "Allocations" "allocations"}}`? I see from the tests that it's intentional.

Reply:
`allocations` is a blank page. Allocations aren't useful to list outside the context of a job or a node.

Relatedly, these breadcrumbs are going to change soon to chain off the job. It was initially this way because we were concerned that it might be disorienting to change the breadcrumb trail to jobs when coming to an allocation from a node, but we recently decided it's nbd.